foundry/applications/frameViewer.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * A simple window application which shows the built documentation pages within an iframe
+ */
+declare class FrameViewer extends Application {
+  constructor(url: string, options: Application.Options);
+
+  url: string;
+
+  /** @override */
+  static get defaultOptions(): Application.Options;
```

Comment:
Could you add a `@defaultValue` with the changed defaults?
Something like:
```ts
/**
 * @override
 * @defaultValue
 * ```
 * mergeObject(super.defaultOptions, {
 *   height: window.innerHeight * 0.9,
 *   width: Math.min(window.innerWidth * 0.9, 1200),
 *   top: (window.innerHeight - height) / 2,
 *   left: (window.innerWidth - width) / 2,
 *   id: "documentation",
 *   template: "templates/apps/documentation.html",
 * })
 * ```
 */
```